frontend/src/pages/article/article-page.tsx

```diff
@@ -40,24 +40,18 @@ const ArticlePage: React.FC = () => {
     return null;
   }
 
-  const MOCKED_TAGS: TagType[] = [
-    { id: 1, name: 'IT' },
-    { id: 2, name: 'CODE' },
-    { id: 3, name: 'Humor' },
-    { id: 4, name: 'Work' },
-    { id: 5, name: 'Tech' },
-  ];
-
   return (
     <Loader isLoading={isLoading} hasOverlay type="circular">
       <Layout>
         <div className={styles.articlePageWrapper}>
-          <ArticleView
-            tags={MOCKED_TAGS}
-            text={article?.text ?? ''}
-            title={article?.title ?? ''}
-            coverUrl={article?.coverUrl ?? ''}
-          />
+          {getArticleStatus === DataStatus.FULFILLED && (
```

Comment:
It seems redundant. We've already done this check earlier (Loader and getArticleStatus === DataStatus.REJECTED)